ui/components/core/src/utils.ts

```diff
@@ -21,3 +21,11 @@ import { schemeSet3 } from "d3-scale-chromatic";
 export function ordinalColorScale(range: Iterable<string>) {
   return scaleOrdinal(schemeSet3).domain(range);
 }
+
```

Comment:
in order to better organize utils, there is a dedicated folder `core/src/lib/utils`. Maybe we could create an index file which would allow to do a `utils.fooFunction`?